src/Token/componentTokens.css

```diff
@@ -0,0 +1,127 @@
+/**/
+[data-color-mode='light'][data-light-theme^='light'],
```

Comment:
Let's just add this to the new Primitives build, I'm not following why you want to avoid doing that? Don't worry about adding it to the old data set, just add it to the new build. I think we can prioritize dotcom for this component since its incredibly product specific, which has access to Primitives CSS variables.

We're working on adding that import to React's Storybook too, it's just slightly complicated. I can try and prioritize that if I have a better sense of your timeline with this PR.

Reply:
I'd love this. The only issue I am having is that we don't know when this is shipped. I am trying to remove interdependencies, so if I add it to v8 I need v8 to be shipped to use it right?